segmentation/utils/spatial_analysis.py

```diff
@@ -0,0 +1,245 @@
+import pandas as pd
+import numpy as np
+import scipy
+import statsmodels
+from statsmodels.stats.multitest import multipletests
+
+# Erin's Data Inputs
+
+# cell_array = pd.read_csv("/Users/jaiveersingh/Downloads/SpatialEn"
+#                          "richment/granA_cellpheno_CS-asinh-norm_revised.csv")
+# marker_thresholds = pd.read_csv("/Users/jaiveersingh/Downloads/Sp"
+#                                 "atialEnrichment/markerThresholds.csv")
+# dist_matrix = np.asarray(pd.read_csv("/Users/jaiveersingh/Documen"
+#                                      "ts/MATLAB/distancesMat5.csv",
+#                                      header=None))
+
+
+def compute_close_cell_num(patient_data, patient_data_markers, thresh_vec,
+                           dist_mat, marker_num, dist_lim, cell_label_idx):
+    """Finds positive cell labels and creates matrix with counts for cells positive for corresponding markers.
```

Comment:
Sometimes functions are hard to explain in just a single line. In that case, it's good to have a single line description, followed by a more in-depth description separated by a full space. I think this function needs a bit more description so people understand what it does